source/includes/steps-starting-vsce-connect-with-advanced-connection-settings.yaml

```diff
@@ -180,52 +196,57 @@ content: |
 
-                 .. note::
+             - id: OIDC
+               name: OIDC
+               content: |
+               
+                 Select :guilabel:`OIDC` if the deployment uses :manual:`OpenID Connect </core/security-oidc/>` 
+                 as its authentication mechanism.
+      
+                 Provide the following information:
 
-                    Starting in MongoDB version 4.0, MongoDB removes
-                    support for the deprecated MongoDB
-                    Challenge-Response (``MONGODB-CR``) authentication
-                    mechanism.
+                 .. list-table::
+                    :header-rows: 1
+                    :widths: 50 50
 
-             - id: scram-sha-256
-               name: SCRAM-SHA-256
-               content: |
-                 Select :guilabel:`SCRAM-SHA-256` if the deployment uses
-                 :manual:`SCRAM-SHA-256 </core/security-scram/>` as its
-                 authentication mechanism (*New in MongoDB 4.0*). If
-                 selected, you must provide the :guilabel:`Username`,
-                 :guilabel:`Password`, and :manual:`Authentication
-                 Database
-                 </core/security-users/#user-authentication-database>`
-                 to authenticate the user.
+                    * - Field
+                      - Description
 
-                 For more information on the ``SCRAM``
-                 authentication mechanism, see :manual:`SCRAM
-                 </core/security-scram/>`.
+                    * - Username
+                      - Optional. OpenID Connect username.
 
-             - id: ldap
-               name: LDAP
-               content: |
+                    * - Auth Code Flow Redirect URI
+                      - Optional. Specify a URI where the identity provider redirects you after authentication.
+                        The URI must match the configuration of the Identity Provider.
+                        The default is ``http://localhost:27097/redirect``.
 
-                 Select :guilabel:`LDAP` if the deployment
-                 uses :manual:`LDAP </core/security-ldap-external/>`
-                 as its authentication mechanism. If selected, you
-                 must provide the :guilabel:`Username` and
-                 :guilabel:`Password` to authenticate
-                 the user.
+                    * - Consider Target Endpoint Trusted
+                      - Optional. Allows connecting to a target endpoint that is not in the 
+                        list of endpoints that are considered trusted by default. Only use 
+                        this option when connecting to servers that you trust.
+
+                    * - Enable Device Authentication Flow
+                      - Optional. When the :ref:`Show Device Auth Flow Checkbox <compass-settings-reference>` 
+                        setting is enabled, |compass-short| can provide you with a URL and code 
```

Comment:
This replacement isn't working in staging, and I don't think we should use |compass-short|. It's unfortunate that our builds don't report this as an error.

Reply:
Doh, yes good catch, this was C+P from Compass due to the fact that now both apps share the same connection form. And yes, reliable logs would be super helpful to spot these sort of things.